posthog/tasks/async_migrations.py

```diff
@@ -59,7 +58,7 @@ def check_async_migration_health() -> None:
 
     # the worker crashed - this is how we find out and process the error
     if migration_instance.celery_task_id not in active_task_ids:
-        if is_current_operation_resumable(migration_instance):
+        if getattr(config, "AUTO_START_ASYNC_MIGRATIONS"):
```

Comment:
are you sure we want this here? If you're removing the resumable concept then we should just try to resume automatically, no? I see `AUTO_START_ASYNC_MIGRATIONS` as a flag that's out of scope here.

Reply:
will do a separate conf